server/Makefile

```diff
@@ -157,6 +157,7 @@ PLUGIN_PACKAGES += mattermost-plugin-zoom-v1.8.0
 PLUGIN_PACKAGES += mattermost-plugin-ai-v0.8.3
 PLUGIN_PACKAGES += focalboard-v8.0.0
 PLUGIN_PACKAGES += mattermost-plugin-msteams-v2.0.3
+PLUGIN_PACKAGES += mattermost-plugin-user-survey-v1.1.0
```

Comment:
This will be bumped to 1.1.1 once the open PR for the bug fix is completed, correct?

Reply:
@marianunez done.